code/datums/hud/critter.dm

```diff
@@ -263,7 +263,7 @@
 						to_pull = tgui_input_list(master, "Which do you want to pull? You can also Ctrl+Click on things to pull them.", "Which thing to pull?", pullable)
 					if(!isnull(to_pull) && BOUNDS_DIST(master, to_pull) == 0)
 						usr = master // gross
```

Comment:
```suggestion
```
This line isn't necessary anymore
and is also awful. christ

Reply:
Line removed